### PR TITLE
Final fixes & documentation

### PR DESCRIPTION
**Pull request by boutell**
## Summary

Final bug fixes and documentation

## What are the specific steps to test this change?

* Follow the README, except **be sure to use the `documentation` branch** of `astro-frontend`
* No linking necessary (depends on this branch of apostrophe-astro)
* starter-kit-astro `main` may be used
* Everything works normally
* README docs feel right for this first release to a technical audience that knows Astro

## What kind of change does this PR introduce?
*(Check at least one)*

- [X] Bug fix
- [X] New feature
- [ ] Refactor
- [X] Documentation
- [X] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [X] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [X] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: components/AposTemplate.astro [hunk 1]
```diff
@@ -1,12 +1,8 @@
 ---
-import { templates } from 'virtual:apostrophe-doctypes'
+import { templates } from 'virtual:apostrophe-doctypes';
 
 const { aposData } = Astro.props;
 
-if (aposData.template === '@apostrophecms/page:notFound') {
-  Astro.response.status = 404;
```
Comment (boutell) [inline components/AposTemplate.astro:7]: Too late. See astro-frontend

File: README.md [hunk 2]
```diff
@@ -179,88 +299,378 @@ It will come with those slots, very related to what we have in the Nunjucks bloc
...
+such as PDFs. For more information, see
+the [attachment field format](https://v3.docs.apostrophecms.org/reference/api/field-formats.html#attachment).
+
+## What to change in your Apostrophe project
+
+Nothing! Well, almost.
+
+* Your project must be using Apostrophe 3.x.
+* You'll need to `npm update` your project to the latest version of `apostrophe`.
+* You'll need to set the `APOS_EXTERNAL_FRONT` environment variable to a secret
```
Comment (BoDonkey) [inline README.md:415]: ...`APOS_EXTERNAL_FRONT_KEY` environment variable

File: README.md [hunk 3]
```diff
@@ -179,88 +299,378 @@ It will come with those slots, very related to what we have in the Nunjucks bloc
...
+[`@astrojs/react` integration](https://docs.astro.build/en/guides/integrations-guide/react/).
+
+In this way, Astro acts as a **univeral bridge** to essentially all modern
+frontend frameworks.
+
+## A note on production use
+
+For production use, any Astro hosting adapter that supports `mode: 'server'` should
+be acceptable. In particular, our [apostrophecms/astro-frontend]
+(https://github.com/apostrophecms/astro-frontend) project comes pre-configured
```
Comment (BoDonkey) [inline README.md:621]: This link is malformed. The space between the brackets and parentheses needs to be removed.

**Review by BoDonkey — APPROVED** [review]
See my comments on the README, but I approved to streamline things.

**Pull request merged**